Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -292,6 +297,20 @@ void ReflectometryReductionOneAuto2::init() {
   setPropertyGroup("Rho", "Polarization Corrections");
   setPropertyGroup("Alpha", "Polarization Corrections");
 
+  // Flood correction
+  propOptions = {"Workspace", "ParameterFile"};
+  declareProperty("FloodCorrection", "Workspace",
+                  boost::make_shared<StringListValidator>(propOptions),
+                  "The way to apply flood correction: "
+                  "Workspace - use FloodWorkspace property to get the flood "
+                  "workspace, ParameterFile - use parameters in the parameter "
+                  "file to construct and apply flood correction workspace.");
+  declareProperty(
+      make_unique<WorkspaceProperty<MatrixWorkspace>>(
+          "FloodWorkspace", "", Direction::Input, PropertyMode::Optional),
+      "A flood workspace to apply. If empty and FloodCorrection is "
```

Comment:
The `.` prevents the tool tip to show the rest of the description. May be better to use a semicolon. Maybe it's ok since you have GUI and nobody opens the auto generated interface.